modules/collections/client/views/collection-edit.html

```diff
@@ -112,36 +111,43 @@ <h1 class="view-title__info--title">
         <fieldset>
           <legend>Main Documents</legend>
 
-          <div class="alert alert-danger" ng-if="!collection.hasPublished">
+          <div class="alert alert-danger" ng-if="validationFlags.documentsTabInvalid">
             <span class="alert__icon glyphicon glyphicon-exclamation-sign"></span>
-            <span class="alert__msg">A minimum of one (1) PUBLISHED Main Document is required before this Collection can be published.</span>
+            <span class="alert__msg">A minimum of one (1) PUBLISHED Main Document is required for a Collection can be published.</span>
```

Comment:
Some grammer weirdness

Reply:
Thank you, I meant to improve it but made it worse. Will fix immediately.